$OpenBSD: patch-src_remminaprefdialog_c,v 1.1 2011/11/09 14:51:42 jasper Exp $

Don't display any SSH options which won't work since we don't use libssh.
(Can't not create the widget as the value is checked, not used, later on, and
this way the patch is kept minimal).

--- src/remminaprefdialog.c.orig	Thu Dec  9 02:51:56 2010
+++ src/remminaprefdialog.c	Wed Nov  9 15:49:26 2011
@@ -427,12 +427,12 @@ remmina_pref_dialog_init (RemminaPrefDialog *dialog)
     priv->scale_quality_combo = widget;
 
     widget = gtk_label_new (_("SSH tunnel local port"));
-    gtk_widget_show (widget);
+    //gtk_widget_show (widget);
     gtk_misc_set_alignment (GTK_MISC (widget), 0.0, 0.5);
     gtk_table_attach_defaults (GTK_TABLE (table), widget, 0, 1, 9, 10);
 
     widget = gtk_entry_new_with_max_length (5);
-    gtk_widget_show (widget);
+    //gtk_widget_show (widget);
     gtk_table_attach_defaults (GTK_TABLE (table), widget, 1, 2, 9, 10);
     g_snprintf (buf, sizeof (buf), "%i", remmina_pref.sshtunnel_port);
     gtk_entry_set_text (GTK_ENTRY (widget), buf);
